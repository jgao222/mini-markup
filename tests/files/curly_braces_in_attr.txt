<style>
    p {
        text-align: center;
    }
</style>

<body>
    <p>
        lorem ispum
    </p>
    <p>
        lorem ipsum
    </p>
</body>