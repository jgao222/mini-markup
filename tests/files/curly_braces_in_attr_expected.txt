<style> {
    p &lbrkt;
        text-align: center;
    &rbrkt;
}

<body> {
    <p> {
        lorem ispum
    }
    <p> {
        lorem ipsum
    }
}